pupa/cli/commands/update.py

```diff
@@ -59,6 +59,46 @@ def save_report(report, jurisdiction):
             )
 
 
+@transaction.atomic
+def save_report(report, jurisdiction):
+    from pupa.models import RunPlan
+    plan = RunPlan.objects.filter(jurisdiction=jurisdiction).latest('id')
+
+    # from pupa.reports.organizations import organization_report
+    from pupa.reports.people import person_report
+    from pupa.reports.posts import post_report
+    from pupa.reports.bills import bill_report
+    from pupa.reports.votes import vote_report
+
+    scrapers = report['plan']['scrapers']
+
+    measures = {}
+
+    # organization_report(jurisdiction)
+
+    if 'people' in scrapers:
```

Comment:
Yup! I could see an edge case where there are none of a particular OCD type, but I don't believe that should be an issue.